package.json

```diff
@@ -2,28 +2,29 @@
   "name": "pokemon-showdown",
   "description": "The server for the Pokémon Showdown battle simulator",
   "version": "0.11.7",
-  "main": ".sim-dist/index.js",
+  "main": "sim/index.ts",
```

Comment:
Like I don't think this actually works in an NPM package.

Reply:
Oh yeah I should probably change that back.